Blockzilla/BrowserViewController.swift

```diff
@@ -547,6 +547,8 @@ class BrowserViewController: UIViewController {
                     self.view.layoutIfNeeded()
                     self.findInPageBar?.becomeFirstResponder()
                 }
+            } else {
+                self.findInPageBar?.becomeFirstResponder()
```

Comment:
This also happens on line 548. Is it possible to move `becomefirstResponder` out of there and just execute it once?

Reply:
Yep I fixed that